packages/stark-ui/src/modules/action-bar/components/action-bar.component.spec.ts

```diff
@@ -24,7 +25,15 @@ describe("ActionBarComponent", () => {
 			providers: [
 				{ provide: STARK_LOGGING_SERVICE, useValue: new MockStarkLoggingService() },
 				{ provide: STARK_ROUTING_SERVICE, useClass: MockStarkRoutingService },
-				TranslateService
+				TranslateService,
+				{
+					// See https://github.com/NationalBankBelgium/stark/issues/1088
+					provide: HAMMER_LOADER,
+					useValue: () =>
+						new Promise(() => {
```

Comment:
Travis is failing because of a violation of the `no-inferred-empty-object-type` linting rule. So I think we can change like this to fix the violation:

`useValue: () => of({}).toPromise()`  or `useValue: () => EMPTY.toPromise()`